src/containers/paint-editor.jsx

```diff
@@ -308,6 +304,56 @@ class PaintEditor extends React.Component {
     setTextArea (element) {
         this.setState({textArea: element});
     }
+    onKeyPress (event) {
```

Comment:
Does this belong in its own HOC?

Reply:
I'm hitting a block doing this because everything depends on everything else

Reply:
This change is quickly becoming massive. I think it should be part of a separate PR.

Reply:
Sounds good